components/templates/NewsPage/NewsHero.js

```diff
@@ -17,9 +17,9 @@ export default function NewsHero({
   if (caption)
     return (
       <Styled.HeroFigure className={className}>
-        <Styled.HeroImageContainer>
+        <Styled.HeroImageContainer data-cy="hero">
           <Styled.HeroImage
-            image={imageData}
+            image={{ ...imageData, priority: true }}
```

Comment:
set high priority on the article heroes since they are the largest contentful paint